spec/models/api/fake_data/hses_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "rails_helper"
+require "fake_issues"
+
+RSpec.describe Api::FakeData::Hses do
+  describe "Issue" do
+    let(:complaint) { FakeIssues.instance.json[:data][4] }
+
+    describe "#request" do
+      let(:issue_endpoint) { Api::FakeData::Hses::Issue.new([:id, complaint[:id]]) }
```

Comment:
[nitpick] I think this array format is difficult to read and reason about, and so I think that readability outweighs having to put an initializer on `Issues`